content/en/platform/corda/5.0/deploying-operating/config-users/permission-system-entities.md

```diff
@@ -0,0 +1,113 @@
+---
+date: '2023-04-24'
+title: "Permission System Entities"
+menu:
+  corda-5:
+    parent: corda-5-cluster-config-users
+    identifier: corda-5-cluster-permission-system-entities
+    weight: 3000
+section_menu: corda-5
+---
+
+The R3's role-based access control (RBAC) permissions system is made up of multiple entities.
+The following diagram presents those entities and relationships between them:
+
+{{< figure src="images/entities.png" figcaption="Permission System Entities" width="55% length="55%">}}
+
+## User
+
+This entity represents a human user or system user.
+
+### Properties
+
+* `Full Name` is a human-readable name. This is a property which will cannot be used to code any permissions against it.
+  It also has a UUID identifier for a concrete reference to the instance of an entity.
+* `Login Name` is the name used for authentication and authorization purposes. It can be in an email address
+  in case of SSO. There can be only one `Enabled` user for a given `Login Name`.
+* `Enabled` is a boolean flag to indicate whether the user is enabled or temporarily suspended.
+* `E-mail` is an email address for the user account used for communication of any important information
+  affecting this account. Same email address may be used for multiple accounts.
+* `Salt value` is a randomly generated value used when hashing the password. It only makes sense when password is
+  assigned as well.
+* `Hashed password` ia a salted and hashed representation of the password.
+ This property can be set to `null` for SSO-only accounts.
+* `Password expiry` is only applicable when password is assigned and when it specifies the timestamp after which the password
+  is deemed expire and needs changing.
+* `Parent Group` is an optional identifier of the group to which the user belongs. This property can be se to `null`. In such
+  case, the user is assumed to belong to the "root" group. In the current model, the user can belong to just a
+  single group. However, groups can be nested.
+
+### Relationships
+
+* `User` can be associated one-to-one with a `Group`.
+* `User` can have multiple roles associated with it.
+* `Change Audit` entries may have one-to-one references to acting `User`.
+* `User` can have multiple `Properties` assigned to it.
+
+### Properties
```

Comment:
Since it is not currently possible to add, remove, modify properties, I would remove this section from the documentation.

Reply:
@vkolomeyko would it be possible for GA?

Reply:
No, it will not be implemented for GA.

Reply:
removed